packages/salesforcedx-vscode-core/src/util/configUtil.ts

```diff
@@ -16,6 +18,9 @@ export enum ConfigSource {
   None
 }
 
+export const defaultUserNameKey = 'defaultusername';
+export const defaultDevHubUserNameKey = 'defaultdevhubusername';
+
```

Comment:
It looks like these two constants were in authInfo originally, did we need to move them to this file?